### DISCUSSION TITLE
Software engineering at google 번역본이 나왔네요~

### DISCUSSION THREAD

**Discussion by TaeHyoungKwon** [ThinkAboutSoftware/AcademicConference#212] [General]
http://mobile.kyobobook.co.kr/showcase/book/KOR/9791162245620?OV_REFFER=https://m.facebook.com/

예전에 이 책으로 우현님이 친구분과 독서토론 모임때 읽으셨다고 했던게 기억나네요ㅎㅎ

사실 잘알려진책이기도하고, 구글에서 검색하면 영문 pdf가 풀려있기도합니다..

어찌되었든 번역본이 나와서, 영문으로볼때 저의 짧은 독해실력으로.. 맥락을 파악할수없었던 부분들을 채워줄거라 기대하는중입니다ㅎㅎ

**Comment by jongfeel** [comment]
이 책이 신간으로 나와서 읽어볼까 하고 장바구니에 넣어 두긴 했는데
우현님이 한번 알려주신 그 책의 번역본이었군요?

장바구니에 넣어 놓은 책은 많은데 지금 책 읽는 속도가 안나고 있어서 ㅋㅋㅋㅋ
올해 안에는 읽기 시작하지 않을까 예상해 봅니다.
